### PR TITLE
openhv 20230917 (new formula)

### PR DESCRIPTION
**Pull request by Mailaender**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [X] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

I can't use system dependencies due to Lua 5.1 being deprecated. https://github.com/Homebrew/homebrew-core/pull/198735.

### CODE REVIEW

File: Formula/o/openhv.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
+class Openhv < Formula
+  desc "Pixelart Science-Fiction Real-Time-Strategy game"
+  homepage "https://www.openhv.net"
+  url "https://github.com/OpenHV/OpenHV.git",
+      using:    :git,
```
Comment (chenrui333) [inline Formula/o/openhv.rb:5]:
```suggestion
```

File: Formula/o/openhv.rb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  license "GPL-3.0-or-later"
+  head "https://github.com/OpenHV/OpenHV.git", branch: "main"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  depends_on "git" => :build
+  depends_on "make" => :build
```
Comment (chenrui333) [inline Formula/o/openhv.rb:17]:
```suggestion
```

File: Formula/o/openhv.rb [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    system "make", "prefix=#{prefix}", "install-executables"
+    system "make", "prefix=#{prefix}", "install-data"
+  end
+
+  test do
+    system "#{bin}/openhv-utility", "--check-missing-sprites"
+    assert_equal 0, $CHILD_STATUS.exitstatus
+
+    system "#{bin}/openhv-utility", "--check-yaml"
+    assert_equal 0, $CHILD_STATUS.exitstatus
```
Comment (chenrui333) [inline Formula/o/openhv.rb:37]:
```suggestion
    system bin/"openhv-utility", "--check-missing-sprites"
    system bin/"openhv-utility", "--check-yaml"
```

**Comment by SMillerDev** [comment]
If this needs to vendor Lua for the exact reason we don't want formula to vendor Homebrew dependencies, I think that makes it a bad fit for Homebrew/core.

https://docs.brew.sh/Acceptable-Formulae#stuff-that-requires-vendored-versions-of-homebrew-formulae

**Comment by Mailaender** [comment]
I understand. Thought that building from source is preferred. Submitted https://github.com/Homebrew/homebrew-cask/pull/193079 instead.

**Pull request closed without merging**